skills/lark-sheets/references/lark-sheets-create-sheet.md

```
@@ -0,0 +1,50 @@
+
+# sheets +create-sheet（创建工作表）
+
+> **前置条件：** 先阅读 [`../lark-shared/SKILL.md`](../../lark-shared/SKILL.md) 了解认证、全局参数和安全规则。
+
+本 skill 对应 shortcut：`lark-cli sheets +create-sheet`。
+
+在已存在的电子表格中新增一个工作表。
+
+> 底层封装的是官方“操作工作表（operate-sheets）”接口。
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)